fixtures/backup/model_dependencies/detailed.json

```diff
@@ -5316,11 +5316,6 @@
         "model": "sentry.environment",
         "nullable": true
       },
-      "owner": {
```

Comment:
Let's leave this in until version 24.6.0 is released for coverage of this case, see my other comment.